frontend/src/v5/ui/routes/dashboard/teamspaces/users/usersList.styles.ts

```diff
@@ -0,0 +1,177 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Cell, Head, Row } from '@/v4/routes/components/customTable/customTable.styles';
+import { FloatingButton, FloatingButtonContainer } from '@/v4/routes/components/floatingActionPanel/floatingActionPanel.styles';
+import { Content as LoadingText } from '@/v4/routes/components/loader/loader.styles';
+import { Container as UserTable, Footer as LicencesFooter } from '@/v4/routes/components/userManagementTab/userManagementTab.styles';
+import { LoaderContainer } from '@/v4/routes/userManagement/userManagement.styles';
+import { PendingInvites } from '@/v4/routes/users/users.styles';
+import styled, { css } from 'styled-components';
+
+const AddUserButton = css`
+	${FloatingButtonContainer} {
```

Comment:
Shouldn't this overrides for v4 be in overrides?

Reply:
I was copying what was done in `routes\dashboard\projects\userPermissions\userPermissions.styles.ts`.
I can instead put it in an overrides file if you think that's better

Reply:
I think that one was before the overrides thing was in created. So yes, if we can lets try to put in there. My guess is that if you place it in overrides, the one from userpermissions is no longer necessary?